numpy/random/generator.pyx

```diff
@@ -3780,8 +3781,6 @@ cdef class Generator:
         >>> arr
         [1 7 5 2 9 4 3 6 0 8] # random
 
-        Multi-dimensional arrays are only shuffled along the first axis:
-
         >>> arr = np.arange(9).reshape((3, 3))
         >>> np.random.Generator().shuffle(arr)
```

Comment:
Add an example of a multidimensional shuffle using axis>0 (or axis=-1, or both).